content/ecosystem/tiers.json

```diff
@@ -4,17 +4,9 @@
     "description": "The main Qiskit packages maintained by IBM Quantum"
   },
   {
-    "name": "Extension",
+    "name": "Extensions",
```

Comment:
should this be a part of this PR?

Reply:
Actually when I tried to build the project on my local machine, some changes happened automatically. I just updated the learn section and added new content but many of them happened automatically.

Reply:
@AanshSavla - out of curiosity, what process/commands did you use to build the project locally?

Reply:
While initially building the project I got this error
error:0308010C:digital envelope routines::unsupported
For that I ran the command:  '''export NODE_OPTIONS=--openssl-legacy-provider'''
At last I ran '''npm run dev'''